.github/workflows/kurtosis.yml

```diff
@@ -0,0 +1,30 @@
+name: Kurtosis sim tests
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *" # Runs at midnight UTC everyday
+  push:
+    branches:
+      - nc/kurtosis
```

Comment:
`push.branches` is added to facilitate review by showcasing the Kurtosis CI run. Will remove it after this PR is approved.